<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectWidgetClass</class>
 <widget class="QWidget" name="SelectWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SelectWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>分类</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="usageCbox">
       <item>
        <property name="text">
         <string>全部</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>饮食</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>出行</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>借贷</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>娱乐</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>教育</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>时间范围</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="beginDateE"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="endDateE"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="selectBut">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="listView"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
